src/picknik_ur_site_config/objectives/re-tare.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Re-tare">
+  <!-- ////////// -->
+  <BehaviorTree ID="Re-tare" _description="Zero the force torque sensors to mitigate accumulated drift error" _favorite="true" _hardcoded="false">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Decorator ID="ForceSuccess">
```

Comment:
You should add in the comment about why we use `ForceSuccess` in this Objective as well

Reply:
Done